catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -140,6 +143,13 @@ define([
         },
         handleClick: function(){
             this.$el.trigger('closeDropdown.'+CustomElements.getNamespace());
+        },
+        serializeData: function(){
+            return {
+                isSearchHistoricalEnabled : properties.isSearchHistoricalEnabled(),
+                /* isSearchArchivedEnabled : properties.isSearchArchivedEnabled(), */
```

Comment:
❓ Any reason this is commented out? Should we uncomment or remove?

Reply:
fixed